src/services/helpdesk/hooks/index.js

```diff
@@ -5,40 +5,79 @@ const hooks = require('feathers-hooks');
 const auth = require('feathers-authentication');
 const restrictToCurrentSchool = globalHooks.ifNotLocal(globalHooks.restrictToCurrentSchool);
 
-function createinfoText(user, category, subject, cloud){
+function createinfoText(user, data){
 	return "Ein neues Problem wurde gemeldet." + "\n"
 	+ "User: " + user + "\n"
-	+ "Kategorie: "+ category + "\n"
-	+ "Betreff: " + subject + "\n"
-	+ "Schauen Sie für weitere Details und zur Bearbeitung bitte in den Helpdesk-Bereich der "+ cloud +".\n\n"
+	+ "Kategorie: "+ data.category + "\n"
+	+ "Betreff: " + data.subject + "\n"
+	+ "Schaue für weitere Details und zur Bearbeitung bitte in den Helpdesk-Bereich der "+ data.cloud +".\n\n"
 	+ "Mit freundlichen Grüßen\n"
-	+ "Deine " + cloud;
+	+ "Deine " + data.cloud;
 }
 
+function createfeedbackText(user, data){
+	let text = "User: " + user + "\n"
+	+ "E-Mail: " + data.email + "\n"
+	+ "Schule: " + data.schoolName + "\n"
+	+ "Bereich ausgewählt: " + data.category + "\n";
+	if (data.desire && data.desire != ""){
+		text = text + "User schrieb folgendes: \n"
+		+ "Als " + data.role + "\n"
+        + "möchte ich " + data.desire + ",\n"
+		+ "um " + data.benefit + ".\n"
+		+ "Akzeptanzkriterien: " + data.acceptanceCriteria;
+	} else {
+		text = text + "User meldet folgendes: \n"
+		+ "Problem Kurzbeschreibung: " + data.subject + "\n"
+		+ "IST-Zustand: " + data.currentState + "\n"
+		+ "SOLL-Zustand: " + data.targetState;
+	}
+	return text;
+}
+
+const problemOrFeedback = hook => {
+	if (hook.data.type === "feedback"){
+		hook.result = {}; //interrupts db interaction
+	}
+	return hook;
+};
+
 exports.before = {
 	all: [auth.hooks.authenticate('jwt')],
 	find: [globalHooks.hasPermission('HELPDESK_VIEW')],
 	get: [globalHooks.hasPermission('HELPDESK_VIEW')],
-	create: [globalHooks.hasPermission('HELPDESK_CREATE'), restrictToCurrentSchool],
+	create: [globalHooks.hasPermission('HELPDESK_CREATE'), restrictToCurrentSchool, problemOrFeedback],
```

Comment:
Better not name it optionaAorB, what if you add more possible options/cases to that function? Get a better name, smth like switchFeedbackType oder denyDbWriteOnType or so.